src/test/java/com/wikia/webdriver/common/driverprovider/NewDriverProvider.java

```diff
@@ -127,14 +125,13 @@ private static EventFiringWebDriver getAndroidInstance() {
   private static EventFiringWebDriver getFFInstance() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getFFInstance" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)